### PR TITLE
Prevent AgentLauncher & Fluent-Bit restarts frequently during normal …

### PR DESCRIPTION
**Pull request by simathih**
…Linux AMA operation

### CODE REVIEW

File: AzureMonitorAgent/agent.py [hunk 1]
```diff
@@ -532,8 +532,11 @@ def enable():
         log_and_exit("Enable", GenericErrorCode, "Failed to add environment variables to {0}: {1}".format(config_file, e))
 
     if "ENABLE_MCS" in default_configs and default_configs["ENABLE_MCS"] == "true":
-        start_amacoreagent()
-        restart_launcher()
+        if platform.machine() != 'aarch64':
+            # enable processes for Custom Logs
+            ensure["azuremonitor-agentlauncher"] = True
+            ensure["azuremonitor-coreagent"] = True
+
```
Comment (hestolz) [inline AzureMonitorAgent/agent.py:539]: there's also line 545 (github not letting me comment there ...) another instance of `start_amacoreagent`, that should probably be changed as well to ensure

File: AzureMonitorAgent/agent.py [hunk 2]
```diff
@@ -532,14 +532,17 @@ def enable():
...
+            ensure["azuremonitor-agentlauncher"] = True
+            ensure["azuremonitor-coreagent"] = True
+            
         # start the metrics and syslog watcher only in 3P mode
         start_metrics_process()
         start_syslogconfig_process()
     elif ensure.get("azuremonitoragentmgr") or is_gcs_single_tenant:
         # In GCS scenarios, ensure that AMACoreAgent is running
-        start_amacoreagent()
+        ensure["azuremonitor-coreagent"] = True
```
Comment (AaronRM) [inline AzureMonitorAgent/agent.py:545]: Need to add a check for aarch64 as in the ENABLE_MCS case.

**Review by AaronRM — CHANGES_REQUESTED** [review]

File: AzureMonitorAgent/agent.py (hunk 1 above)
Comment (hestolz) [reply AzureMonitorAgent/agent.py:539]: linter we have in packaging script would have found it 👍

File: AzureMonitorAgent/agent.py (hunk 2 above)
Comment (simathih) [reply AzureMonitorAgent/agent.py]: added a check

**Review by hestolz — APPROVED** [review]

**Review by AaronRM — APPROVED** [review]

**Pull request merged**